app/src/main/kotlin/com/hippo/ehviewer/ui/screen/DownloadsFilterState.kt

```diff
@@ -0,0 +1,21 @@
+package com.hippo.ehviewer.ui.screen
+
+import androidx.compose.runtime.saveable.Saver
+import androidx.compose.runtime.saveable.listSaver
+import com.hippo.ehviewer.dao.DownloadInfo
+import com.hippo.ehviewer.util.containsIgnoreCase
+
+data class DownloadsFilterState(val label: String?, val state: Int = -1, val keyword: String = "") {
+    companion object {
+        val Saver: Saver<DownloadsFilterState, *> = listSaver(
```

Comment:
Consider make `DownloadsFilterState` a `Parcelable`and annotate with `Parcelize` to avoid handwritten `Saver`

Reply:
I intentionally chose not to use `Parcelize` to reduce codegen overhead because this class is so simple.
But I guess the difference is negligible and using `Parcelize` makes it look cleaner.